server/cluster.go

```diff
@@ -744,6 +746,8 @@ func (sess *Session) rpcWriteLoop() {
 		sess.unsubAll()
 	}()
 
+	heartBeat := time.NewTimer(clusterSessionCleanup)
```

Comment:
Can you add a short comment here mentioning that this is needed for GCing orphaned sessions?